web/jquery.tablesorter.js

```diff
@@ -0,0 +1,1031 @@
+/*
```

Comment:
please include a version number inside javascript file name
this will terribly help when upgrading it, since it will break most of caches, if the version is ommited a lot of browser caches will still use old js
js caching is simply broken on browsers, versioning js filenames is a good way to fight this